core/src/main/kotlin/com/patrykandpatryk/liftapp/core/viewmodel/SavedStateHandleViewModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.patrykandpatryk.liftapp.core.viewmodel
+
+import androidx.lifecycle.SavedStateHandle
+
+interface SavedStateHandleViewModel {
+    val savedStateHandle: SavedStateHandle
+}
```

Comment:
Potential detekt error.
```suggestion
}

```